src/actions.js

```diff
@@ -104,6 +122,11 @@ export function fetchTasks(modulesManager, params) {
   return graphql(payload, ACTION_TYPE.SEARCH_TASKS);
 }
 
+export function fetchTaskHistory(modulesManager, params) {
+  const payload = formatPageQueryWithCount('taskHistory', params, TASK_HISTORY_FULL_PROJECTION());
```

Comment:
To be sure, do we need any kind of params validation here?

Reply:
I think no, there is no need.